proto/prysm/v1alpha1/validator-client/web_api.proto

```diff
@@ -168,10 +168,10 @@ service Auth {
     }
 }
 
-// Type of key manager for the wallet, either direct, derived, or remote.
+// Type of key manager for the wallet, either derived, local, or remote.
 enum KeymanagerKind {
     DERIVED = 0;
-    IMPORTED = 1;
+    LOCAL = 1;
```

Comment:
The problem is the web UI uses IMPORTED in post requests to the backend (this backend server found in validator/rpc), such as in the CreateWallet API endpoint. If we change this here, we will break the web UI. Maybe we can mark this field as deprecated, add LOCAL as an enum value, and handle both cases (imported or local) to not break existing users of the web UI

Reply:
ack. as per offline discussion, we are reverting this rename.